### ISSUE TITLE
Add Comment indicator

### ISSUE THREAD

**Issue by TechSupport26** [lightningrodlabs/acorn#205]
1. There is no indication to show new or unread comments
2. The number of comments does not count correctly with variations between each running Acorn instance, we are experiencing this at around 100 comments.
3. Each comment could have a number to give it some kind of identity.
4. The chat box does not auto-scroll to the latest comment and will sometimes make it part way showing the author of the comment.

**Comment by pegahvaezi** [comment]
Yep there is a lot of work to be done for the comments and notifications features. For now we will be focusing on fixing bugs on the existing functionalities in this area (e.g. #4). 

#2 might be relating to Holochain @Connoropolous ?
